.github/workflows/chromatic.yml

```diff
@@ -1,9 +1,28 @@
 name: "Chromatic"
 
-on: push
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ready_for_review, opened, synchronize]
```

Comment:
I wonder if this is wise.

1. We should at least add `develop` as a branch we check for when pushing. It is the current main branch. Also: What about `release/*` branches?
2. What happens if someone creates a PR against a branch not covered by any of the above? Will Chromatic have a proper baseline to test against?